docs/templates/explanation-template.md

```diff
@@ -0,0 +1,52 @@
+
+For example, you could start an explanation introduction paragraph about computers like the following:
+
+"Computers are a digital machine that is used to perform operations, both arthimetic and logical, automatically. They can be useful in a variety of situations, such as to type a document, code software, or watch a video. This guide is for users who are new to computers and explains what computers are,their history, the benefits of using a computer, and drawbacks of using computers."
+
+## First context section <!-- Header optional if there's only one section in the article -->
+
+The first section should be about the main idea of the topic. The title for a explanation section generally starts with a noun. Examples include "History of computers", "Computer benefits", and "Computers and the internet". 
+
+For example, if you were writing about computers, you could go into more detail about what they do, how they work, and so on. This would expand on the information that was in the introduction paragraph. 
+
+### Subsection with more details <!-- Optional --> 
+
+Subsections go into more detail about some aspect of the functionality, concept, or bigger picture context of the topic. The title for a explanation section generally starts with a noun. Examples include "History of computers", "Computer benefits", and "Computers and the internet".
+
+For example, if you were writing about the larger context of computers in the main section, you could write about the history of computers in this section or the history of a particular computer.
+ 
+Keep in mind that H2 is the highest header level a topic can have. If you are using nested headers, make sure that they're sequential. An H2 can contain an H3, but not the other way around.
+ 
+## Second context section <!-- Optional -->
+
+Additional sections contain more information about another element of your topic. You could write about the benefits and drawbacks of your topic or you could write about how your topic interacts with another part of the software in depth. The title for a explanation section generally starts with a noun. Examples include "History of computers", "Computer benefits", and "Computers and the internet".
+
+For example, if you were writing about computers, this section could cover the benefits of using a computer.
+
+"Using a computer has the following benefits:
+
+* Complete tasks in a faster amount of time than if you were completing them manually
+* Surf the internet"
+
+## See also <!-- Optional -->
+
+This section should include a list of tutorials or how-to guides that a user can visit to extend their learning from this explanation article.
+
+See the following examples of explanation documentation:
+* [Routes](/gateway/{{page.kong_version}}/key-concepts/routes/)
+* [Hybrid-mode](/gateway/{{page.kong_version}}/production/deployment-topologies/hybrid-mode/)
+* [Plugin ordering](/gateway/{{page.kong_version}}/kong-enterprise/plugin-ordering/)
```

Comment:
Since these pages aren't going to be published by netlify, the version variable won't work. I imagine you also probably want to link to the live site, not the markdown source files, so I would put the full URLs here.

```suggestion
* [Routes](https://docs.konghq.com/gateway/latest/key-concepts/routes/)
* [Hybrid-mode](https://docs.konghq.com/gateway/latest/production/deployment-topologies/hybrid-mode/)
* [Plugin ordering](https://docs.konghq.com/gateway/latest/kong-enterprise/plugin-ordering/)
```